src/app/_services/api/admin/category-stats.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
This file needs to be formatted.

Reply:
This file was deleted in reformatting, so no need to.